src/components/Exploration/CardsExplorer.tsx

```diff
@@ -52,20 +55,33 @@ export function CardsExplorer({
   )
   // is the search loading ?
   const [searchLoading, setSearchLoading] = useState<boolean>(false)
+
   // get cardSize from scope or use default
   const cardSize = useMemo<number>(
-    () => (!cardSizeScope ? DEFAULT_SIZE : settings.cardSize),
-    [settings.cardSize, cardSizeScope]
+    () =>
+      !cardSizeScope || !settings.cardSize[cardSizeScope]
+        ? DEFAULT_SIZE
+        : settings.cardSize[cardSizeScope],
+    [cardSizeScope, settings.cardSize]
```

Comment:
Since card size is now provided by scope we check if we have a card size saved or use default size